content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -0,0 +1,143 @@
+
+```bash
+spin deploy       
+Uploading todo-app-v2 version 0.1.0-r6e577b18 to Fermyon Cloud...
+Deploying...
+App "todo-app-v2" accesses a database labeled "default"
+Would you like to link an existing database or create a new database?:
+> Use an existing database and link app to it
+  Create a new database and link the app to it
+```
+
+We again use the up/down arrow keys to make our selection and hit `return` to continue:
+
+```bash
+Which database would you like to link to todo-app-mkii using the label "default":
+> inspiring-otter
+```
+
+In the example here we have chosen to link `todo-app-v2` to our pre-existing `inspiring-otter` Cloud database using via the "default" label, that we added to our `todo-app-v2` application manifest (`spin.toml`) file.
+
+Now we have two seperate Spin applications (`todo-app` and `todo-app-v2`) using their own labels to link to the same database. 
+
+Links and labels are application scoped. Whenever a specific Spin application (say `todo-app`) needs to access the database resource (`inspiring-otter`), it will do so via the label "data".
+
+The Spin application does not need to know the true name of the Cloud resource (`inspiring-otter` database) that it is interacting with. 
+
+A label is understood by all the components within a single application, but the labels applied do not hold any significance to other Spin applications. 
+
+> At this time, only NoOps SQL DB supports labels. However, in the future other Fermyon Cloud resources such as Key Value Store will support links and labels as well. 
+
+## Benefits of Links and Labels
+
+Benefits of the Fermyon Clould links and labels include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
+* **Flexible Lifecycle Management**: Manage Fermyon Cloud resources independently of your Spin application.
+* **Seamless Cloud Integration**: Cloud experience smoothly integrates with local Spin development, no runtime configuration changes required. 
+* **Dynamic App Resource Configuration**: Change the data resource your application points to without rebuilding it.
+
+## Working With Links and Labels
+
+Whenever you add a string to your NoOps SQL database entry in the component manifest, you're creating a label. When you deploy your Spin application to Fermyon Cloud, you will be prompted to link the label to a specific NoOps SQL database instance. Later on, if you'd like to update the link you can do so via the `spin cloud link` command. 
+
+To see your existing labels, you will need to use the `spin cloud` plugin along with the subcommand of the resource you've linked your Spin application to (via a label). For example, if you would like to see the labels you created for your NoOps SQL DB resources, you would run the following command:
```

Comment:
Rather than saying "use the spin cloud plugin" maybe "use the spin cloud command"?

```suggestion
To see your existing labels, run the `spin cloud <resource-type> list` command, where `<resource-type>` is the subcommand for the resource whose links you want to see. For example, if you want to see how your applications and your NoOps SQL databases are linked, run the following command:
```